common/views/components/SearchForm/SearchForm.tsx

```diff
@@ -14,17 +13,11 @@ import { ParsedUrlQuery } from 'querystring';
 import TextInput from '@weco/common/views/components/TextInput/TextInput';
 import ButtonSolid from '@weco/common/views/components/ButtonSolid/ButtonSolid';
 import { trackGaEvent } from '@weco/common/utils/ga';
-import SearchFilters from '@weco/common/views/components/SearchFilters';
-import Select from '@weco/common/views/components/Select/Select';
 import Space from '@weco/common/views/components/styled/Space';
-import SelectUncontrolled from '@weco/common/views/components/SelectUncontrolled/SelectUncontrolled';
```

Comment:
Looks like SelectUncontrolled isn't being used anywhere else, should we clean it up while we're here?